frontend/apps/reader/readerui.lua

```diff
@@ -382,6 +382,9 @@ function ReaderUI:init()
         })
     end
 
+    -- Allow others to change settings based on external factors
+    self:handleEvent(Event:new("DocSettingsLoad", self.doc_settings, self.document.file))
```

Comment:
Isn't `self.document.file` unnecessarily limiting?

Reply:
not for my purposes, however i will change it for anyone else who wants to extend this based on some other heuristic